src/webgpu/api/validation/resource_usages/texture/in_render_common.spec.ts

```diff
@@ -18,4 +18,75 @@ subresources, one for same draw/same pass/different pass, one for visibilities).
 import { makeTestGroup } from '../../../../../common/framework/test_group.js';
 import { ValidationTest } from '../../validation_test.js';
 
-export const g = makeTestGroup(ValidationTest);
+class F extends ValidationTest {
+  getColorAttachment(
+    texture: GPUTexture,
+    textureViewDescriptor?: GPUTextureViewDescriptor
+  ): GPURenderPassColorAttachment {
+    const view = texture.createView(textureViewDescriptor);
+
+    return {
+      view,
+      loadValue: { r: 1.0, g: 0.0, b: 0.0, a: 1.0 },
```

Comment:
API changed. instead use:
```suggestion
      clearValue: { r: 1.0, g: 0.0, b: 0.0, a: 1.0 },
      loadOp: 'clear',
```